Users/userModels.js

```diff
@@ -0,0 +1,34 @@
+var Schema = mongoose.Schema;
```

Comment:
This file should be named ```user-models.js```: https://google.github.io/styleguide/jsguide.html#file-name